src/content/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-monitor.mdx

```diff
@@ -292,5 +292,5 @@ On a typical deployment, migrating from Azure API polling to the Azure monitor i
 <Callout variant="important">
   If both integrations are enabled for the same resource type, you may see duplicate metrics. This can happen if you use a generic name field to filter your results that can be shared from the Azure monitor and the Azure Polling integrations. The results from multiple resources could be joined unintentionally.
 
-  Note that metrics can still be differentiated by faceting using the field `collector.name`. Also note that rate limits as enforced by Azure will be shared between any current polling integrations and the Azure monitor.
+  Note that metrics can still be differentiated by faceting using the field `collector.name`. `collector.name` may not always be available on an API polling resource, its absense an indication of no Azure Monitor coverage. Also note that rate limits as enforced by Azure will be shared between any current polling integrations and the Azure monitor.
```

Comment:
Suggestion: its absence is an indication of no Azure monitor coverage.

Reply:
Thanks for checking this out.

If we add the "an" article, to my ear it feels like we need an "and" in the second clause.

So it could be:
`collector.name` may not always be available on an API polling resource, its absence an indication of no Azure Monitor coverage. (original)

or

`collector.name` may not always be available on an API polling resource, and its absence is an indication of no Azure Monitor coverage. (adjusted)

Reply:
`collector.name` may not always be available on an API polling resource, and its absence is an indication of no Azure monitor coverage. (This is better)

Reply:
Sounds great!